tests/e2e/specs/shopper/checkout-different-addresses.test.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	openDocumentSettingsSidebar,
+	uiUnblocked,
+} from '@woocommerce/e2e-utils';
+
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	shopper,
+	preventCompatibilityNotice,
+	reactivateCompatibilityNotice,
+} from '../../../utils';
+
+import { BILLING_DETAILS, SHIPPING_DETAILS } from '../../../utils/constants';
+const SIMPLE_PRODUCT_NAME = '128GB USB Stick';
+
+const config = require( 'config' );
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'Skipping Checkout tests', () => {} );
+
+describe( 'Shopper → Checkout → Can have different shipping and billing addresses', () => {
+	beforeAll( async () => {
+		// prevent CartCheckoutCompatibilityNotice from appearing
+		await preventCompatibilityNotice();
+		await merchant.login();
+
+		// Activate all address fields.
+		await visitBlockPage( 'Checkout Block' );
+		await openDocumentSettingsSidebar();
+		await selectBlockByName(
+			'woocommerce/checkout-shipping-address-block'
+		);
+		await expect( page ).toClick( 'label', { text: 'Company' } );
+		await expect( page ).toClick( 'label', {
+			text: 'Apartment, suite, etc.',
+		} );
+		await expect( page ).toClick( 'label', { text: 'Phone' } );
+		await saveOrPublish();
+	} );
+
+	afterAll( async () => {
```

Comment:
We've added a product to the cart. To avoid leaving any side effects from our tests, let's remove it:

```js
afterAll( async () => {
  await shopper.block.emptyCart();
}
```

Reply:
> Isn't the cart empty given that we place the order? I decided not to run the empty cart command to save time.

@nielslange, this is true if our test is successful. But, if our test fails (e.g., the order didn't go through), our product will not get removed from the cart, which may affect the following tests.

Reply:
> @nielslange, this is true if our test is successful. But, if our test fails (e.g., the order didn't go through), our product will not get removed from the cart, which may affect the following tests. Please let me know if you think this is something worth considering. Otherwise, feel free to ignore this suggestion.

I see your point and just added `await shopper.block.emptyCart();` both to `beforeAll()` and `afterAll()`.